### PR TITLE
Creating OpenstackCinderVolumeStuck Alert

### PR DESCRIPTION
**Pull request by rajivmucheli**
@auhlig 

Please review the Cinder Alert definition and help in configuring the cinder alerts to openstack-cinder channel to slack.

### CODE REVIEW

File: system/kube-monitoring/charts/prometheus-frontend/openstack-cinder.alerts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+### Cinder Alerts ###
+
+ALERT OpenstackCinderVolumeStuck
+  IF (sum(openstack_stuck_volumes_count_gauge)) by (region,host) *60) > 0
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-cinder.alerts]: Hi @rajivmucheli, there are 2 brackets behind `..gauge))`. Remove one. Please check the expression in prometheus beforehand. Also`region` in the by clause is optional. 
I couldn't find the metric you're trying to alert on. Is it supposed to be there yet?

**Comment by auhlig** [comment]
From syntax perspective this looks good now. But since the metric doesn't exist currently it's hard to say if the alert condition makes sense. Maybe @fwiesel can tell.

**Comment by fwiesel** [comment]
The metric will only appear, when there is actually a stuck value.

**Comment by auhlig** [comment]
Yes. I didn't get it. The `FOR 5m` would indicate that the condition needs to apply for exactly that time.

**Comment by rajivmucheli** [comment]
@auhlig i presumed *60 had to be part of the syntax. I have removed *60 now, if the syntax looks good, can i or you merge this PR ? Could you plz add the changes the on how to redirect cinder alerts to openstack-cinder slack channel ?

**Comment by jobrs** [comment]
@rajivmucheli can we have a project_id label on the metric? Also I wonder what the _gauge suffix is good for. I believe unlike "_counter" for counters it is not common to add a "_gauge" suffix. See https://prometheus.io/docs/practices/naming/

**Comment by fwiesel** [comment]
Counters are monotonously increasing. I think, [Gauge](https://prometheus.io/docs/concepts/metric_types/#gauge) is correct. It will hopefully decrease again.

**Comment by jobrs** [comment]
absolutely, but you do not have to add the "_gauge" suffix. That's somewhat default.

**Comment by rajivmucheli** [comment]
@auhlig @jobrs good to Merge now ?

**Comment by jobrs** [comment]
Ok, I see you guys are talking only about the alert. 

I also wanted to create some awareness for the metric naming and the `project_id` label for those metrics which can be connected to a project.

So probably my comments are more relevant for @fwiesel. Fabian, would you be able to provide a project_id label for this metric? In which repo can I find the corresponding exporter?

**Comment by fwiesel** [comment]
The metric comes from here: https://github.com/sapcc/openstack-helm/blob/master/cinder/values.yaml#L63-L80

The `_gauge` suffix is also added by the statsd exporter.

**Comment by jobrs** [comment]
Not necessarily: 

> If -statsd.add-suffix is set, the exporter appends the metric type (_gauge,
_counter, _timer) to the resulting metrics. This is enabled by default for
backward compatibility but discouraged to use. Instead, it is better to
explicitly define the full metric name in your mapping and run the exporter
with -statsd.add-suffix=false

But since the suffix is there now and it does not do any harm probably we should keep it.

The other thing is more important: Can we have a project_id label on this metric. I am not aware of the DB schema, that's why I am asking

**Comment by fwiesel** [comment]
Sure, I've added it. Ideally, that metric is always empty and we hopefully do not have to expose it to our customers.

**Comment by jobrs** [comment]
agreed

**Pull request merged**